src/libaktualizr/package_manager/androidmanager.h

```diff
@@ -0,0 +1,39 @@
+#ifndef ANDROIDMANAGER_H
+#define ANDROIDMANAGER_H
+
+#include "package_manager/packagemanagerinterface.h"
+#include "storage/invstorage.h"
+
+class AndroidInstallationDispatcher;
+
+class AndroidManager : public PackageManagerInterface {
+ public:
+  explicit AndroidManager(std::shared_ptr<INvStorage> storage) : storage_(std::move(storage)) {}
+  ~AndroidManager() override = default;
+  std::string name() const override { return "android"; }
+  Json::Value getInstalledPackages() const override;
+
+  Uptane::Target getCurrent() const override;
+  bool imageUpdated() override { return true; };
+
+  data::InstallOutcome install(const Uptane::Target &target) const override;
+  data::InstallOutcome finalizeInstall(const Uptane::Target &target) const override;
+
+ private:
+  std::shared_ptr<INvStorage> storage_;
+};
+
+struct AndroidInstallationState {
+  enum State { STATE_NOP, STATE_READY, STATE_IN_PROGRESS, STATE_INSTALLED, STATE_UPDATE };
```

Comment:
Another style consideration: we now prefer using `enum class` and name enum variants according to google's c++ style.

e.g.: https://github.com/advancedtelematic/aktualizr/blob/97214dcab4acc2284d4169503ae09fabefe6af34/src/libaktualizr/primary/initializer.h#L11